RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/26 [00:00<?, ?it/s]  4%|▍         | 1/26 [00:47<19:35, 47.02s/it] 12%|█▏        | 3/26 [00:51<05:21, 13.97s/it]100%|██████████| 26/26 [00:51<00:00,  1.99s/it]
Exception ignored in: <function Pool.__del__ at 0x2ba072cf8160>
Traceback (most recent call last):
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
